doc/source/examples/multiphysics/static-bubble/static-bubble.rst

```diff
@@ -0,0 +1,335 @@
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Analytical solution
+    #---------------------------------------------------
+
+    subsection analytical solution
+      set enable                = true
+      set verbosity             = quiet
+      set filename              = L2Error
+      subsection uvwp
+       set Function expression = 0; 0; if (x^2 + y^2 < 0.5^2 , 2, 0)
+      end
+    end
+
+---------------------------
+Running the simulation
+---------------------------
+
+Call the gls_navier_stokes_2d by invoking:
+
+``mpirun -np 8 gls_navier_stokes_2d static-bubble.prm``
+
+to run the simulation using eight CPU cores. Feel free to use more.
+
+
+.. warning::
+    Make sure to compile lethe in `Release` mode and
+    run in parallel using mpirun. This simulation takes
+    :math:`\approx` 10 mins on 8 processes.
+
+-----------
+Results
+-----------
+
+Using Paraview, we can visualize the evolution of the velocity field over the time:
+
++------------------------------------------+------------------------------------------+
+| t = 0 s                                  | t = 0.1 s                                |
+|                                          |                                          |
+| .. image:: images/static-bubble-t0p0.png | .. image:: images/static-bubble-t0p1.png |
++------------------------------------------+------------------------------------------+
+| t = .5 s                                 | t = 3.0 s                                |
+|                                          |                                          |
+| .. image:: images/static-bubble-t1p5.png | .. image:: images/static-bubble-t3p0.png |
++------------------------------------------+------------------------------------------+
+
+The time evolution of the :math:`\mathcal{L}^2` norm of the error on the velocity magnitude is obtained from a Gnuplot script available in the example folder by launching in the same directory the following command:
+
+.. code-block:: text
+
+  gnuplot -c "./postprocess.gnu" "./output"
+
+where ``./postprocess.gnu`` is the path to the provided script and ``./output`` is the path to the directory that contains the ``L2Error.dat`` file. The figure, named ``L2Error.png``, is outputted in the directory ``./output``.
+
+.. image:: images/L2Error.png
+
+""""""""""""""""""""""""""""""""
+Mesh convergence study
+""""""""""""""""""""""""""""""""
+
+While the filters presented in section :ref:`Normal and curvature computations` allow to decrease the magnitude of the spurious currents, it can be seen from the previous results that they don't completely disappear. It is therefore interresting to see if they vanish with a mesh refinement by performing a space convergence study on their magnitude.
```

Comment:
```suggestion
While the filters presented in section:ref:`Normal and curvature computations` allow to decrease the magnitude of the spurious currents, it can be seen from the previous results that they don't completely disappear. It is, therefore, interesting to see if they vanish with a mesh refinement by performing a space convergence study on their magnitude.
```